docs/.vitepress/theme/index.ts

```diff
@@ -22,9 +20,9 @@ if (inBrowser) {
 }
 
 export default {
-  extends: DefaultTheme,
+  extends: VoidZeroTheme,
   Layout() {
-    return h(DefaultTheme.Layout, null, {
+    return h(VoidZeroTheme.Layout, null, {
       'home-features-after': () => h(HomePage),
     })
```

Comment:
I think after the theme change `HomePage` (docs/.vitepress/components/HomePage.vue) isn't not used any more?